static/js/components/EmploymentForm.js

```diff
@@ -287,6 +294,12 @@ class EmploymentForm extends ProfileFormFields {
       ui: { workHistoryEdit }
     } = this.props;
     let cardClass = workHistoryEdit ? '' : 'profile-tab-card-greyed';
+    let cardBody = this.renderCardBody();
+
+    if (!cardBody) {
```

Comment:
it will be true in case:

- when user is viewing employment on profile
- when user is viewing his own profile
- when user is viewing other user profile with data

It will be false when.
- user is viewing other learner profile and their is no data